index.html

```diff
@@ -34,22 +38,212 @@ <h1>Hey There.
+                        </div>
+                        
+                    </div>
+            </div>
+
+            <div class="card-works">
+                <div class="place-hold">
+                    <div class="r-b">
+                        <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+                    <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+                    <ul class="btn-hold">
+                        <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                        <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                        <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+                    </ul>
+                    </div>
+                    <div class="actz">
+                        <button class="clicks btnn">See Project</button>
+                    </div>
+                    
+                </div>
+        </div>
+
+
+        <div class="card-works">
+            <div class="place-hold">
+                <div class="r-b">
+                    <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+                <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+                <ul class="btn-hold">
+                    <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                    <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                    <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+                </ul>
+                </div>
+                <div class="actz">
+                    <button class="clicks btnn">See Project</button>
+                </div>
+                
+            </div>
+    </div>
+
+    <div class="card-works">
+        <div class="place-hold">
+            <div class="r-b">
+                <div class="title-post"><h3 class="ghg">Profesional Art Printing Data</h3></div>
+            <div class="supportt"><h6 class="sup-txt">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard</h6></div>
+            <ul class="btn-hold">
+                <li class="html"><a href="#" class="txt-ht">HTML</a></li>
+                <li class="bootss"> <a href="#" class="b-txt"> bootstrap</a> </li>
+                <li class="rub"><a href="#" class="rub-txt">Ruby</a></li>
+            </ul>
+            </div>
+            <div class="actz">
+                <button class="clicks btnn">See Project</button>
+            </div>
+            
+        </div>
+</div>
+          </div>
+    </div>   
+
+
+    
+    
+        </section>
+
+        <!--about myself-->
+        <section class="about">
```

Comment:
Please ensure that the content of the `about section` is aligned with flexbox according to the project requirement.